### DISCUSSION TITLE
Possibility to use Mypal as a base

### DISCUSSION THREAD

**Discussion by louies0623** [rmottola/Arctic-Fox#199] [Ideas]
Forgive me for not being an English user. First of all, I have difficulty using Arctic-Fox. It is not a language I can use and is not supported when using the palemoon language pack. However, I also have this problem when using mypal, but as long as I add firefox The official language xpi can be used. Secondly, some Internet pages, such as mypal, run well and have pdf support. So is it possible to make a mac version based on mypal?

**Comment by rmottola** [comment]
I only work on ArcticFox, I did not know which languages pack do you use, it is possible I broke something, I never investigated that thing.
MyPal is now a quite different fork on a different code base apparently up to esr68 mozilla, I have no idea if that can compile on which version on mac, I bet it needs rust. You better ask mypal project if they are interested.

**Comment by rmottola** [comment]
For developments or mac versions of another browser, please discuss with the other project.
If there are specific patches that you feel could be of help, we can see if they can be ported over.

PDF support was added back to ArcticFox and is decently working.

**Comment by rmottola** [comment]
Currently, no language packs exists for ArcticFox, help implementing them is welcome.
MyPal is another browser, refer to them. As a development reference for ArcticFox we continue to use Gecko and TenFourFox.
